fundingmanager.go

```diff
@@ -1933,37 +1928,32 @@ func (f *fundingManager) sendFundingLocked(peer lnpeer.Peer,
 	// server has had the time to connect to the peer. We keep trying to
 	// send fundingLocked until we succeed, or the fundingManager is shut
 	// down.
+	peerKey := completeChan.IdentityPub
 	for {
 		fndgLog.Debugf("Sending FundingLocked for ChannelID(%v) to "+
-			"peer %x", chanID,
-			completeChan.IdentityPub.SerializeCompressed())
+			"peer %x", chanID, peerKey.SerializeCompressed())
 
-		if peer == nil {
-			peer, err = f.cfg.FindPeer(completeChan.IdentityPub)
+		if peer != nil {
+			err = peer.SendMessage(false, fundingLockedMsg)
 			if err != nil {
-				continue
+				fndgLog.Warnf("Unable to send fundingLocked "+
+					"to peer %x: %v",
+					peerKey.SerializeCompressed(), err)
+			} else {
```

Comment:
Can `peer` be `nil` here? If not, this can be written
```go
err = sendMessage(..)
if err == nil {
    break
}
print(err)
... go on ...
```

EDIT: see other comment about `nil` peer